CarePartner/Views/BottomTrayView.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  BottomTrayView.swift
+//  CarePartner
+//
+//  Created by Nathaniel Hamming on 2023-06-14.
+//  Copyright © 2023 Tidepool Project. All rights reserved.
+//
+
+import SwiftUI
+
+protocol TrayContent: View {
+    var discoverableContentHeight: CGFloat { get }
+}
+
+struct BottomTrayView<Content: TrayContent>: View {
```

Comment:
Shouldn't we be using the built in swiftUI `.sheet()` modifier?  You can have custom detents, and set modality with `.presentationBackgroundInteraction(.enabled(upThrough: .medium))`

Reply:
Thank! I needed to move the min target to 16.4 to support the needed behaviour. Also checking with design about the slight style changes.